knowledge-base/tooltip-position-rotated-div.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Adjusting Tooltip Position for Rotated Divs in Blazor
+description: Learn how to adjust the position of the Telerik Blazor Tooltip when the target element is rotated, ensuring it displays correctly.
+type: how-to
+page_title: How to Correct Tooltip Position on Rotated Elements in Blazor
+slug: tooltip-kb-position-rotated-div
+tags: tooltip, blazor, css
+res_type: kb
+ticketid: 1657431
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>
+                Grid for Blazor, <br />
+                Tooltip for Blazor
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+When rotating div elements that have an associated [Tooltip]({%slug tooltip-overview%}) to display additional information, the Tooltip might not align correctly with the target element. The goal is to adjust the Tooltip's position so it aligns appropriately with the center of the rotated div, and optionally, remove the Tooltip's callout pointer for a cleaner appearance.
+
+This KB article also answers the following questions:
+- How to correctly position a Tooltip on a rotated div in Blazor?
+- How can I hide the callout pointer of a Tooltip in Blazor?
+- Is there a way to dynamically adjust Tooltip positioning based on element rotation in Blazor?
+
+## Solution
+
+To address the issue of Tooltip misalignment on rotated div elements, consider the following two approaches:
```

Comment:
Don't use a sentence in a parent section to introduce other sections. This approach is valid for tables and lists, not for sections. To solve the issue, add a list with two items where each item links to the corresponding approach.